index.html

```diff
@@ -78,6 +78,165 @@ <h1 class="heading">
+                <li class="tile-2">CSS</li>
+                <li class="tile-3">Javascript</li>
+                <li class="tile-4">Html</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+
+          <div class="project-tile">
+            <div class="box-3 project-tile">
+              <h1 class="tile-heading">
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h1>
+              <ul class="tile-ul">
+                <li class="tile-1">Ruby on rails</li>
+                <li class="tile-2">CSS</li>
+                <li class="tile-3">Javascript</li>
+                <li class="tile-4">Html</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+
+          <div class="project-tile">
+            <div class="box-4 project-tile">
+              <h1 class="tile-heading">
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h1>
+              <ul class="tile-ul">
+                <li class="tile-1">Ruby on rails</li>
+                <li class="tile-2">CSS</li>
+                <li class="tile-3">Javascript</li>
+                <li class="tile-4">Html</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+
+          <div class="project-tile">
+            <div class="box-5 project-tile">
+              <h1 class="tile-heading">
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h1>
+              <ul class="tile-ul">
+                <li class="tile-1">Ruby on rails</li>
+                <li class="tile-2">CSS</li>
+                <li class="tile-3">Javascript</li>
+                <li class="tile-4">Html</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+
+          <div class="project-tile">
+            <div class="box-6 project-tile">
+              <h1 class="tile-heading">
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h1>
+              <ul class="tile-ul">
+                <li class="tile-1">Ruby on rails</li>
+                <li class="tile-2">CSS</li>
+                <li class="tile-3">Javascript</li>
+                <li class="tile-4">Html</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <!-- About Me -->
+      <section>
+        <div class="about-me">
+          <h1 class="heading">About Me</h1>
+          <p class="description">
+            I can help you build a product,feature or website look through some
+            of my work and experience! If you like what you see and have a
+            project you need coded, don't hesitate to contact me.
+          </p>
+          <button>Get My Resume</button>
+        </div>
```

Comment:
- [ ] The button (e.g., "Get my Resume") should follow the button styling: Active, Hover, and Pressed. Kindly implements it. See the design for reference
### Design
![figma](https://user-images.githubusercontent.com/81727988/185511996-0f414913-62b0-4aa0-a94e-a45894223143.png)